STM32Cube/Core/Src/main.c

```diff
@@ -219,6 +219,7 @@ int main(void)
 
   /* USER CODE BEGIN RTOS_EVENTS */
   /* add events, ... */
+  for(int i = 0; i <10000; i++){} //delay boot to allow millis_ timer to stabilize
```

Comment:
Did this fix the "it doesnt work the first time the board is programmed"?

Reply:
tested, I'm having trouble even replicating the issue? I've removed the line